### ISSUE TITLE
Transcoding or decoding multi-PID heic images

### ISSUE THREAD

**Issue by gorinje** [gpac/gpac#2845]
Although gpac handled well heic images, and gives extensive documentation on image extraction, no information is given to reconstruct a multi-pid images in heic into a single image. Example of these images can be found
[here](https://github.com/tigranbs/test-heic-images/tree/master), e.g. [image1.heic](https://github.com/tigranbs/test-heic-images/raw/master/image1.heic).

It seems from the documentation that only extraction of single items can be currently done.

**Comment by jeanlf** [comment]
This is correct, we currently don't support HEIF grid reconstruction or other multi-pids images.  Any help on this is welcome, as we don't plan on supporting this in the near future...
